src/bundle.ts

```diff
@@ -32,14 +36,22 @@ export class BuildBundler extends Transform {
   bundler: Bundler;
   sharedFile: File;
 
+  files = new Map<string, File>();
+
   constructor(config: ProjectConfig, analyzer: BuildAnalyzer) {
     super({objectMode: true});
 
     this.config = config;
+
     this.analyzer = analyzer;
     this.sharedBundleUrl = 'shared-bundle.html';
     this.bundler = new Bundler({
-      analyzer: analyzer.analyzer,
+
+      // TODO(usergenic): Creating a new Analyzer with a blank cache is going
+      // to mean, at least, a doubling of analysis efforts for bundling phase.
+      // Ideally we would fork existing analyzer and replace its urlLoader if
+      // there were an affordance to do so.
+      analyzer: new Analyzer({urlLoader: new StreamLoader(this)}),
```

Comment:
I'm not 100% where that `file-changed?` logic should live yet, but I think `_fork()` may need API change in `Analyzer` and `AnalysisContext` to support.

- Added https://github.com/Polymer/polymer-analyzer/issues/547 "Extend the `Analyzer#_fork()` API to support replacing the `urlLoader` option (possibly replace any of the initialization options?)"
- Added https://github.com/Polymer/polymer-analyzer/issues/548 "Can Analyzer/AnalysisCache analyze in a mode where it tests for file content changes?"